README.md

```diff
@@ -251,6 +251,8 @@ import DeviceInfo from 'react-native-device-info';
 | [hasNotch()](#hasNotch)                           | `boolean`           |  ✅  |   ✅    |   ✅    | 0.23.0 |
 | [isLandscape()](#isLandscape)                     | `boolean`           |  ✅  |   ✅    |   ✅    | 0.24.0 |
 | [getDeviceType()](#getDeviceType)                 | `string`            |  ✅  |   ✅    |   ❌    | ?      |
+| [isAutoDateAndTime()](#isAutoDateAndTime)         | `string`            |  ❌  |   ✅    |   ❌    | ?      |
+| [isAutoTimeZone()](#isAutoTimeZone)               | `string`            |  ❌  |   ✅    |   ❌    | ?      |
```

Comment:
oops - 2 changes actually, this is a boolean return isn't it? same as isAirplaneMode